Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,45 @@
-#from django.shortcuts import render
+from lxml import etree
 
-# Create your views here.
+from django.http import HttpResponse
+from django.views.generic import ListView
+
+from .models import Product
+
+
+class ProductListView(ListView):
```

Comment:
I am thinking it should simply inherit from View. The base one, since the whole logic to create XML is generated anyway. 
https://akademiait.com.pl/class-based-views-w-django-wygodny-sposob-tworzenia-widokow-czesc-1/

Reply:
Done.